iohub/multipagetiff.py

```diff
@@ -292,8 +294,19 @@ def _set_dtype(self):
         tf = TiffFile(self._files[0])
 
         self.dtype = tf.pages[0].dtype
+        xml = tf.ome_metadata
+        # assuming X and Y pixel sizes are the same
+        xy_size = re.search(r"(?<=PhysicalSizeX=\")[\d\.\d]+", xml)
```

Comment:
Do you know which datasets have `PhysicalSizeX` as a metadata field? This might be a device-specific field. 

I might suggest transferring `PixelSizeUm` instead (see attached screenshot). This field is present in the existing test dataset (though it's set to 0), and it's present if you acquire an MDA with MM's demo camera. 


<img width="463" alt="Screenshot 2023-05-03 at 12 13 05 PM" src="https://user-images.githubusercontent.com/9554101/235976215-a05669e7-c579-46e3-9866-e96e68b0511a.png">

Reply:
Also, `PixelSizeUm` uses MM's `> Device > Pixel Size Calibration` feature which is very convenient.

Reply:
> Do you know which datasets have `PhysicalSizeX` as a metadata field?

I got this from the test datasets generated from the demo config.

Reply:
OK it's in MM's custom tag. Do note that this tag is also not very stable though.

Reply:
Also `0.0` is not a valid scale transform so those datasets will still be set to 1 (identity).

Reply:
> OK it's in MM's custom tag. Do note that this tag is also not very stable though.

Roger that. I think it's appropriate for us to prioritize micromanager's tags, since that's what we're using internally (and for us OME-TIFF ~= MicroManager multipage TIFF). 

Do you have reason to believe this tag is unstable across MM releases?

> Also 0.0 is not a valid scale transform so those datasets will still be set to 1 (identity).

Great!

Reply:
Great! Thanks a bunch for this PR, @ziw-liu.